utils/sync-results.sh

```diff
@@ -33,3 +34,8 @@ do
     echo "${loc} already exists and is not a link, delete or move it to create a link."
   fi
 done
+
+# zip and sync example results
+zip -r core_example_results.zip $repo_base/example-results
+aws s3 cp core_example_results.zip $s3_base/example_results.jpg --acl public-read
```

Comment:
```suggestion
aws s3 cp core_example_results.zip $s3_base/core_example_results.zip --acl public-read
```
This should be `zip` not `jpg`, also let's name it with the full folder name and the core appended to it. I think my thought process here is that we would keep separate zip files with the output from just the core workflow and then each module, otherwise users might get confused seeing all the other output together.

Reply:
Ah yes, that makes total sense 👍